Makefile

```diff
@@ -1,6 +1,7 @@
 DOCKERHOST = docker.io
 DOCKERORG = import-vm-apb
 IMAGENAME = import-vm
+QUAY_ORG = kubevirt
```

Comment:
@fabiand Will you be able to configure build process in Quay so anytime we merge (after this PR) we will rebuild the image?